server/Makefile

```diff
@@ -157,7 +157,7 @@ MMCTL_PACKAGES=$(shell $(GO) list ./... | grep -E 'server/v8/cmd/mmctl')
 TEMPLATES_DIR=templates
 
 # Plugins Packages
-PLUGIN_PACKAGES ?= mattermost-plugin-antivirus-v0.1.2
+PLUGIN_PACKAGES ?= mattermost-plugin-antivirus-v1.0.0
```

Comment:
I've made a mistake in https://github.com/mattermost/mattermost-marketplace/pull/351. Antivirus doesn't work on Cloud. Still, it's fine to update the packaged version.